server/protocols/jmap-rfc-8621/src/test/scala/org/apache/james/jmap/SerializerTest.scala

```diff
@@ -0,0 +1,196 @@
+      |      "maxObjectsInSet": ${MAX_OBJECTS_IN_SET.value},
+      |      "collationAlgorithms": [
+      |        "$ALGO_1",
+      |        "$ALGO_2",
+      |        "$ALGO_3"
+      |      ]
+      |    },
+      |    "${MAIL_IDENTIFIER.asString()}": {
+      |      "maxMailboxesPerEmail": ${MAX_MAILBOXES_PER_EMAIL.get.value},
+      |      "maxMailboxDepth": ${MAX_MAILBOX_DEPTH.get.value},
+      |      "maxSizeMailboxName": ${MAX_SIZE_MAILBOX_NAME.value},
+      |      "maxSizeAttachmentsPerEmail": ${MAX_SIZE_ATTACHMENTS_PER_EMAIL.value},
+      |      "emailQuerySortOptions": [],
+      |      "mayCreateTopLevelMailbox": true
+      |    }
+      |  },
+      |  "accounts": {
+      |    "${USER_1_ID.value}": {
+      |      "name": "${USER_1.asString}",
+      |      "isPersonal": true,
+      |      "isReadOnly": false,
+      |      "accountCapabilities": {
+      |        "${CORE_IDENTIFIER.asString()}": {
+      |          "maxSizeUpload": ${MAX_SIZE_UPLOAD.value},
+      |          "maxConcurrentUpload": ${MAX_CONCURRENT_UPLOAD.value},
+      |          "maxSizeRequest": ${MAX_SIZE_REQUEST.value},
+      |          "maxConcurrentRequests": ${MAX_CONCURRENT_REQUESTS.value},
+      |          "maxCallsInRequest": ${MAX_CALLS_IN_REQUEST.value},
+      |          "maxObjectsInGet": ${MAX_OBJECTS_IN_GET.value},
+      |          "maxObjectsInSet": ${MAX_OBJECTS_IN_SET.value},
+      |          "collationAlgorithms": [
+      |            "$ALGO_1",
+      |            "$ALGO_2",
+      |            "$ALGO_3"
+      |          ]
+      |        }
+      |      }
+      |    },
+      |    "${USER_2_ID.value}": {
+      |      "name": "${USER_2.asString}",
+      |      "isPersonal": false,
+      |      "isReadOnly": false,
+      |      "accountCapabilities": {
+      |        "${CORE_IDENTIFIER.asString()}": {
+      |          "maxSizeUpload": ${MAX_SIZE_UPLOAD.value},
+      |          "maxConcurrentUpload": ${MAX_CONCURRENT_UPLOAD.value},
+      |          "maxSizeRequest": ${MAX_SIZE_REQUEST.value},
+      |          "maxConcurrentRequests": ${MAX_CONCURRENT_REQUESTS.value},
+      |          "maxCallsInRequest": ${MAX_CALLS_IN_REQUEST.value},
+      |          "maxObjectsInGet": ${MAX_OBJECTS_IN_GET.value},
+      |          "maxObjectsInSet": ${MAX_OBJECTS_IN_SET.value},
+      |          "collationAlgorithms": [
+      |            "$ALGO_1",
+      |            "$ALGO_2",
+      |            "$ALGO_3"
+      |          ]
+      |        }
+      |      }
+      |    }
+      |  },
+      |  "primaryAccounts": {
+      |    "${MAIL_IDENTIFIER.asString()}": "${USER_1_ID.value}",
+      |    "${CONTACT_IDENTIFIER.asString()}": "${USER_2_ID.value}"
+      |  },
+      |  "username": "${USER_1.asString}",
+      |  "apiUrl": "$URL",
+      |  "downloadUrl": "$URL",
+      |  "uploadUrl": "$URL",
+      |  "eventSourceUrl": "$URL",
+      |  "state": "$STATE"
+      |}""".stripMargin
```

Comment:
That's a rather long json... I was wondering if it's possible to do like we do in Java, putting a big JSON like this in a resource file and loading it in the test?

Reply:
done, and I will rebase